shared/src/main/java/com/couchbase/lite/Array.java

```diff
@@ -6,22 +6,19 @@
 import java.util.ArrayList;
 import java.util.ConcurrentModificationException;
 import java.util.Date;
-import java.util.HashMap;
 import java.util.Iterator;
 import java.util.List;
-import java.util.Map;
 
 import static com.couchbase.lite.internal.support.ClassUtils.cast;
 
 /**
  * Array provides access to array data.
  */
-public class Array extends ReadOnlyArray implements ArrayInterface, ObjectChangeListener, FleeceEncodable {
+public class Array extends ReadOnlyArray implements ArrayInterface, FleeceEncodable {
     //---------------------------------------------
     // member variables
     //---------------------------------------------
     private List<Object> list = null;
-    Map<ObjectChangeListener, Integer> changeListeners = new HashMap<>();
     private boolean changed = false;
     private int changedCount = 0;
```

Comment:
`changedCount` is still used by Iterator.
https://github.com/couchbase/couchbase-lite-android/blob/c029de8091bb931b1a53d2073ab954ca9b996904/shared/src/main/java/com/couchbase/lite/Array.java#L348-L368